routes/create-or-update-file.js

```diff
@@ -86,6 +76,7 @@ module.exports = function(config) {
 
         response.on('end', function() {
           body = JSON.parse(body);
+          delete body.buffer;
```

Comment:
This makes me think that we shouldn't bother sending the buffer back when a new file is created.